www/src/components/marketplace/MarketplaceRepositories.js

```diff
@@ -30,6 +35,8 @@ const filterTokenStyles = {
   backgroundColor: 'fill-one-selected',
```

Comment:
Line 33 above this needs to be removed. It was an old hack to match the filter token heights to the text field, and is now causing them to be too tall.

Reply:
This line: `minHeight: '42px',`